core/src/mender-artifact.c

```diff
@@ -122,8 +122,14 @@ static mender_err_t artifact_read_data(mender_artifact_ctx_t *ctx, mender_artifa
 /**
  * @brief Process chunk of artifact data
  */
-static mender_err_t process_artifact_data_callback(
-    const char *type, const cJSON *meta_data, const char *filename, size_t size, void *data, size_t index, size_t length, mender_artifact_download_data_t *dl_data);
+static mender_err_t process_artifact_data_callback(const char                      *type,
+                                                   const cJSON                     *meta_data,
+                                                   const char                      *filename,
+                                                   size_t                           size,
+                                                   void                            *data,
+                                                   size_t                           index,
+                                                   size_t                           length,
+                                                   mender_artifact_download_data_t *dl_data);
```

Comment:
The formatting (here and below) belongs to an earlier commit ;)

Reply:
Good catch!